MoviesMVP/MoviesMVPTests/NetworkServiceTests.swift

```diff
@@ -0,0 +1,123 @@
+// NetworkServiceTests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import XCTest
+
+@testable import MoviesMVP
+
+/// Network service tests
+final class NetworkServiceTests: XCTestCase {
+    // MARK: - Public properties
+  
+    var networkService: NetworkServiceProtocol?
```

Comment:
```suggestion
    var networkService: NetworkServiceProtocol?
```
private
это и ниже 3 свойства